app/cdap/components/DirectiveInput/Components/AutoComplete/index.tsx

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, Typography } from '@material-ui/core';
+import MyDataPrepApi from 'api/dataprep';
+import {
+  IAutoCompleteProps,
+  IOnRowClickValue,
+} from 'components/DirectiveInput/Components/AutoComplete/types';
+import { defaultFuseOptions } from 'components/DirectiveInput/constants';
```

Comment:
dont keep them separate, directly give in state

Reply:
Done